src/components/App.js

```diff
@@ -116,9 +105,10 @@ class AppComponent extends React.Component {
     const { loadingBundleUrl, installerData, selectedSpec, fetchingInstallerDataError } = this.state;
     const { isMobile } = this.props;
     const sha = this.props.sha;
-    const bundleUrl = `curl ${process.env.API_URL}/${sha} | sudo bash`
+    const bundleUrl = `curl -LO ${process.env.KURL_BUNDLE_URL}/${sha} | sudo bash`
```

Comment:
i think bundleUrl is a bad name here for this variable. perhaps rename it to installerUrl and revert this change

```suggestion
    const installerUrl = `curl -LO ${process.env.API_URL}/${sha} | sudo bash`
```

Reply:
@jala-dx this wasn't resolved